components/AppFooter.vue

```diff
@@ -20,18 +20,12 @@ const providers: ProviderName[] = ['twitter', 'discord', 'github']
           {{ $formatMessage('footer_docs_text') }}
         </a>
         <a
-          href="https://lukso.network/faq"
+          href="https://support.lukso.network/"
           class="nav-apax-12-medium-uppercase text-purple-41 hover:text-purple-31 hover:underline"
           target="_blank"
         >
           {{ $formatMessage('footer_faq_text') }}
```

Comment:
Ill put the "Need help?" text here rather than the FAQ text

Reply:
`footer_need_help_text`

Reply:
done